### PR TITLE
[ROC-2475] new macro for radio group with show hide hint panel

### PR DESCRIPTION
**Pull request by comanirudha**
Notes:
*
*
*

### CODE REVIEW

File: macros/form.njk [hunk 1]
```diff
@@ -231,6 +231,39 @@
...
+        label = option.label,
+        name = name,
+        form = form,
+        value = option.value,
+        checked = option.checked or (selection === option.value),
+        bold = option.bold
+        ) }}
+        {% if option.hint %}
+          <div class="panel panel-border-narrow js-hidden" id="{{name}}-{{option.value}}" aria-hidden="false">
+          <p>{{ option.hint }}</p>
```
Comment (timja) [inline macros/form.njk]: can you indent this please

File: macros/form.njk [hunk 2]
```diff
@@ -231,6 +231,39 @@
   </div>
 {% endmacro %}
 
+{% macro radioGroupWithHintPanel(
```
Comment (timja) [inline macros/form.njk:234]: you will need to bump the version in this project in order to publish it

Comment (comanirudha) [reply macros/form.njk:234]: done, new version is 1.18.2

Comment (timja) [reply macros/form.njk:234]: it should be 1.19.0 as you've added a new feature 👍 
.2 would be for fixing a bug

**Review by timja — APPROVED** [review]

**Pull request merged**